### DISCUSSION TITLE
Ability to draw on the PDF using pens! (Ex: Surface Pens and other 1st and 3rd Party pens)

### DISCUSSION THREAD

**Discussion by Wizard451345** [sumatrapdfreader/sumatrapdf#2584] [Ideas]
This is a very useful program that I literally just found out about it, and it looks great! My favorite is that you can change the text and page color which is very useful when reading in the dark! :D 

Regardless I am sure that many Windows users have and probably will have touch screen capabilities in the future. I believe that at least being able to annotate without a mouse and keyboard could be important, especially for those who are taking notes on it on the computer. Could you allow the option to draw (via mouse, touch, or pen/ stylus) just like if it were Microsoft Edge? 

Thank you all for your hard work!!

**Comment by GitHubRulesOK** [comment]
Coverd by https://github.com/sumatrapdfreader/sumatrapdf/issues/961 [arrows, lines, etc using mouse paths]
